libs/application/template-api-modules/src/lib/modules/templates/signature-collection/signature-list-signing/signature-list-signing.service.ts

```diff
@@ -90,6 +94,10 @@ export class SignatureListSigningService extends BaseTemplateApiService {
         throw new TemplateApiError(errorMessages.maxReached, 405)
       }
     }
+    if (lists.length === 0) {
+      throw new TemplateApiError(errorMessages.active, 405)
```

Comment:
If lists.length === 0, does that mean there is nothing found or does that reperesent that a user is not allowed to see them?

I don't see 405s that often, so mostly just curious :)

Reply:
This should be a 404, but it does not mean that nothing exists hust thtat there is nothing currently open for the user.